extensions/wikia/AdEngine3/src/slots.js

```diff
@@ -405,8 +405,13 @@ export default {
 	},
 
 	injectIncontentPlayer() {
```

Comment:
This method does two things: injecting the player and returning boolean telling us if the player has been injected or not. How about splitting it into two methods: `isIncontentPlayerInjected()` and `injectIncontentPlayer()`? Then the first one can be used in the setup when setting up `'custom.hasIncontentPlayer'` in the context and in the second method when we decide if we need to execute `setupIncontentPlayerForDistroScale()` 🤔

Reply:
Discussed. Let's move forward 🤷‍♂️

Once UCP goes live and replaces Oasis this method will be replaced with a cleaner one in terms of responsibilities ;)